requirements.txt

```diff
@@ -1,3 +1,7 @@
+matplotlib==3.8.0
+numpy==1.24.4
 pandas==2.2.3
-matplotlib==3.9.2
-numpy==2.1.2
\ No newline at end of file
+#torch==2.2.2
+torch==2.4.1
```

Comment:
I also have some error saying that this version is not available for python environments yet, which is a problem because we do want to use python environments. How much of an issue is to use the other version?

Reply:
yeah i think we can use the previous one as well